### PR TITLE
fix tiny-universe\content\TinyTransformer\readme.md

### PR DESCRIPTION
**Pull request by CliPg**
感觉原文有一点小问题，小k应该只表示一个key的词向量吧。因此对原文做了一点小修改，如有误，请指出！（此外原文line96的latex渲染好像优点问题）

### CODE REVIEW

**Review by linhongyu510 — APPROVED** [review]
The correction is mathematically sound and makes the paragraph internally consistent.

For one query `q ∈ R^(1×d_k)` and three key vectors, the keys must be packed into a matrix `K ∈ R^(3×d_k)`. Then:

```text
q @ K.T ∈ R^(1×3)
```

produces one similarity score per key. A single key vector `k ∈ R^(d_k)` would instead produce only one scalar, so the original lowercase `k = [...]` sentence conflicts with the following `qK^T` expression.

I cross-checked this against:

- the original Transformer paper, which explicitly says that keys are packed together into matrix `K` before computing `QK^T`: https://arxiv.org/html/1706.03762#S3.SS2.SSS1
- PyTorch's reference implementation, which computes `query @ key.transpose(-2, -1)` and gives key shape `(..., S, E)`: https://docs.pytorch.org/docs/stable/generated/torch.nn.functional.scaled_dot_product_attention.html

A minimal numerical check with `q.shape == (1, 2)` and `K.shape == (3, 2)` produced `scores.shape == (1, 3)` and scores `[1, 2, 3]`, while `q @ k.T` for one key produced one value.

I also verified that:

- current `main` still contains the lowercase-`k` mismatch
- the PR introduces matrix `K` before `qK^T` and removes the duplicated explanation
- the change applies conflict-free to current `main` and `git diff --check` passes
- no competing open or historical PR addresses this notation

The branch is 31 commits behind `main`, but GitHub reports `MERGEABLE/CLEAN` and the merge-tree check succeeds. No CI checks or commit statuses are reported.

Non-blocking follow-up: the next equation still mixes uppercase function arguments with lowercase symbols, `attention(Q,K,V) = softmax(qK^T)v`. A future cleanup could make the single-query form consistently `attention(q,K,v)` (and explicitly show `K` as row-stacked), before introducing the fully batched `Q,K,V` form. That inconsistency predates this PR and does not invalidate the correction here.
